dev/io.openliberty.jakarta.websocket.2.1/bnd.bnd

```diff
@@ -0,0 +1,34 @@
+
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.jakarta.websocket.2.1; singleton:=true
+
+javac.source: 11
+javac.target: 11
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
+
+Export-Package: \
+	jakarta.websocket.server; version="2.1.0"
+
+Include-Resource: \
+  @${repo;jakarta.websocket:jakarta.websocket-api;2.1.0}!/META-INF/LICENSE, \
+  @${repo;jakarta.websocket:jakarta.websocket-api;2.1.0}!/META-INF/NOTICE
+
+instrument.disabled: true
+
+publish.wlp.jar.suffix: dev/api/spec
+
+-buildpath: \
+	jakarta.websocket:jakarta.websocket-api;version=2.1.0
```

Comment:
Is there a module-info that we need to exclude?

Reply:
No need as it's not included in the first place.